api/graphql/models/album.go

```diff
@@ -17,6 +17,7 @@ type Album struct {
 	Owners   []User `gorm:"many2many:user_albums;constraint:OnDelete:CASCADE;"`
 	Path     string `gorm:"not null"`
 	PathHash string `gorm:"unique"`
+	CoverID  int    `gorm:"index;"`
```

Comment:
Is there a reason why this is indexed?

Also it might be better to have the type be `int*` to indicate that it can be null, and then update all null checks to be `album.CoverID == nil` rather than `album.CoverID == 0`, I think tha's more clear.

Reply:
No particular reason, I just copied some of the previous code without thinking! Explicit null checks are a good idea, agreed on that.